### DISCUSSION TITLE
Added custom domain but don't receive mails anymore

### DISCUSSION THREAD

**Discussion by BassamxMednini** [vercel/vercel#8563] [Help]
I went through the following [instructions](https://vercel.com/guides/how-do-i-add-a-custom-domain-to-my-vercel-project) and everything works beautifully. Stupidly, I no longer receive any emails. Now I found [this article](https://vercel.com/guides/why-has-email-stopped-working) where it says that I need to add MX records. Unfortunately I haven't found the information neither on Google nor at Namecheap. My domain, as well as hoster/mail provider runs everything via Namecheap. Does anyone have experience and could help me with this?

**Comment by amyegan** [comment]
Hi @BassamxMednini! If you changed your nameservers to connect the custom domain, then you would need to add MX records as described in the article you linked. In that case, Namecheap's [Private Email](https://www.namecheap.com/support/knowledgebase/article.aspx/1340/2176/namecheap-private-email-records-for-domains-with-thirdparty-dns/) article might help. If you used A and CNAME records (without modifying nameservers), then your email should not have been affected and you may need to [contact Namecheap for help](https://www.namecheap.com/help-center).
